### ISSUE TITLE
Panasonic Lumix DMC-G70 not detected

### ISSUE THREAD

**Issue by departet** [gphoto/gphoto2#612]
**Describe the bug**

Pretty much the same as [here][(url](https://github.com/gphoto/gphoto2/issues/596)). BUT i do use WSL with usb passthrough. The camera gets detected when running lsusb.
I ran the command while setting up photobooth 4.

michael@DESKTOP-P6P1GFE:/mnt/c/Windows/system32$ sudo lsusb
Bus 002 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
Bus 001 Device 002: ID 04da:2370 Panasonic (Matsushita) DMC-G70
Bus 001 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub

michael@DESKTOP-P6P1GFE:/mnt/c/Windows/system32$ gphoto2 --auto-detect
Model                          Port
----------------------------------------------------------

michael@DESKTOP-P6P1GFE:/mnt/c/Windows/system32$ sudo gphoto2 --capture-image
*** Error ***
Could not detect any camera
*** Error (-105: 'Unknown model') ***

**Name the camera**
Panasonic Lumix DMC-G70

**libgphoto2 and gphoto2 version**
gphoto2            2.5.27         gcc, popt(m), exif, cdk, aa, jpeg, readline
libgphoto2         2.5.27         standard camlibs, gcc, ltdl, EXIF
libgphoto2_port 0.12.0         iolibs: disk ptpip serial usb1 usbdiskdirect usbscsi, gcc, ltdl, EXIF, USB, serial without locking

**To Reproduce**
Attach camera in pc connection mode or in ptp mode. Same result with both modes.

**Comment by hfiguiere** [comment]
and pretty much the same answer https://github.com/gphoto/gphoto2/issues/596#issuecomment-1722445721

tl;dr try the latest version of libgphoto2 first.

**Comment by departet** [comment]
When installing from source it excludes lumix. Is this needed when using ptp mode? If so, how do i get it to include lumix. Linux noob here...
`libgphoto2      2.5.31         standard camlibs (SKIPPING docupen lumix), gcc, no ltdl, EXIF`

**Comment by msmeissn** [comment]
> When installing from source it excludes lumix. Is this needed when using ptp mode? If so, how do i get it to include lumix. Linux noob here... `libgphoto2 2.5.31 standard camlibs (SKIPPING docupen lumix), gcc, no ltdl, EXIF`

No, the lumix driver is only for the Lumix WIFI mode. The USB Lumix stuff is in the ptp2 driver.

Can you attach lsusb -v output for the camnera in PTP mode?

**Comment by departet** [comment]
Same result as before with the current version
```
Bus 001 Device 002: ID 04da:2370 Panasonic (Matsushita) DMC-G70
Device Descriptor:
  bLength                18
  bDescriptorType         1
  bcdUSB               2.00
  bDeviceClass            0
  bDeviceSubClass         0
  bDeviceProtocol         0
  bMaxPacketSize0        64
  idVendor           0x04da Panasonic (Matsushita)
  idProduct          0x2370
  bcdDevice            1.00
  iManufacturer           1 Panasonic
  iProduct                2 DMC-G70
  iSerial                 0
  bNumConfigurations      1
  Configuration Descriptor:
    bLength                 9
    bDescriptorType         2
    wTotalLength       0x0027
    bNumInterfaces          1
    bConfigurationValue     1
    iConfiguration          0
    bmAttributes         0xc0
      Self Powered
    MaxPower                2mA
    Interface Descriptor:
      bLength                 9
      bDescriptorType         4
      bInterfaceNumber        0
      bAlternateSetting       0
      bNumEndpoints           3
      bInterfaceClass       255 Vendor Specific Class
      bInterfaceSubClass    160
      bInterfaceProtocol    176
      iInterface              0
      Endpoint Descriptor:
        bLength                 7
        bDescriptorType         5
        bEndpointAddress     0x01  EP 1 OUT
        bmAttributes            2
          Transfer Type            Bulk
          Synch Type               None
          Usage Type               Data
        wMaxPacketSize     0x0200  1x 512 bytes
        bInterval               0
      Endpoint Descriptor:
        bLength                 7
        bDescriptorType         5
        bEndpointAddress     0x82  EP 2 IN
        bmAttributes            2
          Transfer Type            Bulk
          Synch Type               None
          Usage Type               Data
        wMaxPacketSize     0x0200  1x 512 bytes
        bInterval               0
      Endpoint Descriptor:
        bLength                 7
        bDescriptorType         5
        bEndpointAddress     0x83  EP 3 IN
        bmAttributes            3
          Transfer Type            Interrupt
          Synch Type               None
          Usage Type               Data
        wMaxPacketSize     0x0200  1x 512 bytes
        bInterval              16
Device Qualifier (for other device speed):
  bLength                10
  bDescriptorType         6
  bcdUSB               2.00
  bDeviceClass            0
  bDeviceSubClass         0
  bDeviceProtocol         0
  bMaxPacketSize0        64
  bNumConfigurations      1
Device Status:     0x0001
  Self Powered
```

**Comment by Kotaku-Nico** [comment]
Hi @departet — I’m researching the same Lumix/photobooth gap on Windows. Did you ever get the G70 working reliably after this issue, or did you end up switching cameras/software or abandoning the setup?

I’m especially curious whether WSL and USB passthrough remained the blocker. Thanks!

**Comment by departet** [comment]
Hi, @Kotaku-Nico we unfortunately did not find a solution to the problem. In the end we got a different camera and a different software (running on windows) which worked right out the box.
